### PR TITLE
Add a New Feature Share Game in Social Media

### PR DESCRIPTION
**Pull request by amryarios**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link**
  **Here you can add your preview link--**
- [Preview Link](http://rawgit.com/amryarios/labyrinth/master/index.html)
- Fixes`#<Issue no.>`
, 

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
[Mention the bug/facilitie solved/improved]

### My steps to solve it,
[Mention the files changed. Add what changes you have done.] 

### Screenshots,
image

- Google+ Share
image

- Twitter
image

- Facebook
image

### CODE REVIEW

**Comment by niccokunzmann** [comment]
@amryarios Thanks, I would like to merge it.
Could you please have a look at the error messages? https://travis-ci.org/fossasia/labyrinth/builds/328083408#L632

They tell us of potential problems. The game broke several times, so we introduced them to keep the game running. What do you think?

**Comment by amryarios** [comment]
image
https://travis-ci.org/fossasia/labyrinth/builds/328083408#L632
I've seen it and check it out, it says there are too many classes? So what do I do? because I follow the code section "Follow us on". then it is said in error that div and ul there is problem? I think it's fine because I followed the "Follow us on" section and just changed the destination url. Give your opinion about what I say :) hope you can understand because I am not good in English. :+1:

File: index.html [hunk 1]
```diff
@@ -84,6 +84,36 @@
 
 				<div class="border1"></div>
 
+                <div class="card-content white-text">
+	                <span class="card-title">Share Game</span>
+				</div>
+				<div class="card-action">
+					<ul class="social-icons">
+						<div class="social-container">
```
Comment (niccokunzmann) [inline index.html]: here, you can switch ul and li
There is no problem in the section below having a div in the ul.
It should look like this;

```
<div> 
  <ul>
       no div here
    <li>
      <div>
```

File: index.html [hunk 2]
```diff
@@ -84,6 +84,36 @@
...
+							</li>
+							<li>
+								<a class="btn-floating transparent waves-effect waves-light twitter" target="_blank" href="https://twitter.com/share?url=http://rawgit.com/fossasia/labyrinth/master/index.html"
+								class="social-icon">
+									<i class="fa fa-twitter"></i>
+								</a>
+							</li>
+							<li>
+								<a class="btn-floating transparent waves-effect waves-light googleplus" target="_blank" href="https://plus.google.com/share?url=http://rawgit.com/fossasia/labyrinth/master/index.html"
+								class="social-icon">
```
Comment (niccokunzmann) [inline index.html]:
```
<a class="btn-floating transparent waves-effect waves-light googleplus" target="_blank" href="https://plus.google.com/share?url=http://rawgit.com/fossasia/labyrinth/master/index.html"
+								class="social-icon">
```
The last line belongs to the first. I fixed it with

```
<a class="btn-floating transparent waves-effect waves-light googleplus social-icon" target="_blank" href="https://plus.google.com/share?url=http://rawgit.com/fossasia/labyrinth/master/index.html">
```

**Comment by niccokunzmann** [comment]
Hi, I think I understand. I gave two comments.

**Comment by niccokunzmann** [comment]
Please also have a look at the other section you claim you copied it from. It was corrected there, too.

File: index.html (hunk 1 above)
Comment (amryarios) [reply index.html]: image
I am confused for this part? Did < li > 94 move to line 92?

**Comment by amryarios** [comment]
image
and this is this error? how to handle it? @niccokunzmann  :+1:

File: index.html (hunk 1 above)
Comment (abishekvashok) [reply index.html]: No.Change div inside the `ul` to `li`.  you can use li within another li

**Comment by amryarios** [comment]
And finally get over all problems :) @niccokunzmann

**Comment by STEdirisuriya** [comment]
[Mention the bug/facilitie solved/improved]

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**